### PR TITLE
Docs: Deploy to AWS with SST

### PR DESCRIPTION
**Pull request by jayair**
- [x] I have read the [Contribution guide](https://github.com/solidjs/solid-docs/blob/main/CONTRIBUTING.md)
- [ ] This PR references an issue (except for typos, broken links, or other minor problems)

### Description(required)

A doc on how to deploy SolidStart to AWS with SST. I tried it locally and it looked good.

### CODE REVIEW

**Review by LadyBluenotes — APPROVED** [review]

**Comment by LadyBluenotes** [comment]
Thank you for this @jayair !

**Review by LadyBluenotes — APPROVED** [review]

**Comment by atilafassina** [comment]
Hello @jayair 

First of all, thank you very much for this contribution.
I just fixed the build step (there was an outdated `pnpm-lock.yaml` breaking things.

I also adjusted some formatting and added tabbed code blocks for `pnpm`, `yarn`, and `bun` on top of `npm`.

I'd like to wait on your go-ahead before merging it, with the build working, you should be able to review it more easily now.

**Comment by jayair** [comment]
Thank you!

Yeah it looks great!

**Review by atilafassina — APPROVED** [review]

**Pull request merged**
